src/data-dictionary/events/NrAiIncidentExternal/aggregationTag.md

```diff
@@ -0,0 +1,10 @@
+---
+name: aggregationTag.*
+type: attribute
+events:
+  - NrAiIncidentExternal
+---
+
+Any trigger events with the `aggregationTag` prefix and the same suffix attribute are grouped together in an incident. A resolve event resets this aggregation.
```

Comment:
This seems a bit confusing to me, but I may just not have enough context. I'm confused by what "same suffix" means here. Would it be `aggregationTag.triggerHere.aggregationTag`?

Reply:
I think I came up with something clearer.